docs/about/whats_new.rst

```diff
@@ -8,30 +8,65 @@ What's New
 Next Version
 ============
 
-- docs: fix broken links :pull:`1725`
-- Update dependencies :pull:`1727`, :pull:`1731`
+v1.9.3 (15th April 2025)
```

Comment:
I think the suggested fix for #1696 should be in this release so `Product.grid_spec` can be undeprecated. Deprecations are like salt when cooking, it's easy to add more later, but hard to un-salt the food.

Reply:
No OWS crash, but ping doesn't work with postgis driver in OWS. #1782 makes the missing method public again.

Reply:
I've merged (and included #1696 but I've left #1782 for now.   Anticipating a 1.9.4 release fairly soon for the dask fixes which I will also note here.